sdk/java/pulumi/src/main/java/com/pulumi/experimental/automation/commands/exceptions/CommandException.java

```diff
@@ -0,0 +1,12 @@
+// Copyright 2025, Pulumi Corporation
+
+package com.pulumi.experimental.automation.commands.exceptions;
+
+import com.pulumi.experimental.automation.commands.CommandResult;
+import com.pulumi.experimental.automation.exceptions.AutomationException;
+
+public class CommandException extends AutomationException {
```

Comment:
I made this a subclass of `AutomationException` for convenience, but thinking about it more now, I'm wondering if it should just subclass Java's `Exception` and if we throw `CommandException` wrap that in an `AutomationException`, e.g. `catch (CommandException e) { throw new AutomationException(e); }`. 

Also now thinking the other exceptions that subclass `CommandException` probably should subclass `AutomationException` directly, and probably not live in `commands.exceptions` but next to wherever `AutomationException` lives.